integration/js/steps/WaitForMeetingCreationAndMeetingReadinessCheckerInitializationStep.js

```diff
@@ -0,0 +1,31 @@
+const {KiteTestError, Status, TestUtils} = require('kite-common');
+const AsyncAppWaitTestStep = require('../utils/AsyncAppWaitTestStep');
+
+class WaitForMeetingCreationAndMeetingReadinessCheckerInitializationStep extends AsyncAppWaitTestStep {
+  constructor(kiteBaseTest, sessionInfo) {
+    super(kiteBaseTest, sessionInfo);
+  }
+
+  static async executeStep(KiteBaseTest, sessionInfo) {
+    const step = new WaitForMeetingCreationAndMeetingReadinessCheckerInitializationStep(KiteBaseTest, sessionInfo);
+    await step.execute(KiteBaseTest);
+  }
+
+  timeoutThresholdInMilliSeconds() {
+    return 10000
```

Comment:
Should we remove this wait then https://github.com/aws/amazon-chime-sdk-js/blob/master/integration/js/steps/StartMeetingReadinessCheckerStep.js#L24?